webui/react/src/components/ExperimentContinueModal.tsx

```diff
@@ -125,9 +116,7 @@ const ExperimentContinueModalComponent = ({
     : [ExperimentCopyMapping, experimentEntityCopyMap];
   const actionCopy = actionCopyMap[modalState.type];
 
-  useEffect(() => setOriginalConfig(experiment.configRaw), [experiment]);
-
-  const requiredFields = useMemo(() => [EXPERIMENT_NAME, MAX_LENGTH], []);
+  const requiredFields = useMemo(() => [EXPERIMENT_NAME], []);
```

Comment:
nit: do we need `requiredFields`?

Reply:
I think it's no harm keeping it, in case we want to expand it in the future

Reply:
consider pulling this and the other array out of the component, given that they are static values.